core/camel-support/src/main/java/org/apache/camel/support/beans/SimpleDataHolderBean.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.support.beans;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SimpleDataHolderBean {
```

Comment:
@rgannu thank you, didn't notice this class in the `org.apache.camel.support.beans` package. However, I am wondering, is this really needed to be in `org.apache.camel.support.beans` package? As I can see here, this can be just moved to the component rabbitmq package, unless there is a reason which I am not aware of?

Reply:
This bean could be used for anything and not just for rabbitmq and hence I moved it inside camel-support module.

Reply:
I am sorry but we do not want to make camel-core / camel-support bigger than what we think is needed. So can you not do that and add the bean inside the rabbitmq component

Reply:
ok. No problem. I will move it to camel-rabbitmq module this bean.